### DISCUSSION TITLE
Display "short_name" instead of "name"

### DISCUSSION THREAD

**Discussion by muralito** [organicmaps/organicmaps#3374] [Ideas]
Hi. As the screen resolution in some cases are small, i think it would be useful, in some situations, to display in street names, the value of "short_name", or to abreviate some words.
Maybe some extreme examples to illustrate teh case, a street in Montevideo, "Presidente General Oscar Diego Gestido", in the street signs is "Pte. Gral. Oscar D. Gestido" (desabreviated in OSM), but the people usually refers to the street as "Gestido". Another one, streer "General José Esteban Brito del Pino" which short name is just "Brito del Pino".

**Comment by biodranik** [comment]
It's a good idea. Can you please show some screenshots to demonstrate the problem?

**Comment by muralito** [comment]
![Screenshot_20220911-210605](https://user-images.githubusercontent.com/7611868/190533677-44b1245c-ddee-4798-b12f-d50ab27013ad.png)
![Screenshot_20220911-210558](https://user-images.githubusercontent.com/7611868/190533683-319bf739-14cd-4666-b826-ebe37e99a6a4.png)
![Screenshot_20220911-210304](https://user-images.githubusercontent.com/7611868/190533684-a9999f94-737a-4050-8cd5-2167284deee8.png)
![Screenshot_20220911-210237](https://user-images.githubusercontent.com/7611868/190533687-14f3f201-6fde-4a44-a9f3-3d145bdab686.png)
![Screenshot_20220911-210144](https://user-images.githubusercontent.com/7611868/190533688-9ad093c9-e4ad-4a49-a9fd-e69671167c96.png)
